src/frontend/src/views/Resources/index.tsx

```diff
@@ -0,0 +1,228 @@
+                - This week Dr. Sidney Bell joined us to discuss antigenic
+                evolution of viruses - that is, how viral surface proteins can
+                change how they &quot;look&quot; to our immune systems, and in
+                some cases eventually escape our immunity.
+              </span>,
+              <span key={10}>
+                <NewTabLink href="https://www.youtube.com/watch?v=lFJ_2G4u8w4">
+                  Week 12
+                </NewTabLink>{" "}
+                - This week I compared and contrasted phylogenetic placements
+                versus phylogenetic trees, and described the different
+                situations when I prefer one over the other.
+              </span>,
+              <span key={11}>
+                <NewTabLink href="https://www.youtube.com/watch?v=lFJ_2G4u8w4">
+                  Week 13
+                </NewTabLink>{" "}
+                - This week we branched out from SARS-CoV-2, and I gave some
+                examples of how you can use genomic epidemiology to investigate
+                the epidemiology of TB.
+              </span>,
+              <span key={12}>
+                <NewTabLink href="https://www.youtube.com/watch?v=lFJ_2G4u8w4">
+                  Week 14
+                </NewTabLink>{" "}
+                - This week the focus was on practical applications of genomic
+                epidemiology, and I walked through some case studies showing
+                different ways in which genomic epidemiology had supported epi
+                investigations in California.
+              </span>,
+              <span key={13}>
+                <NewTabLink href="https://www.youtube.com/watch?v=lFJ_2G4u8w4">
+                  Week 15
+                </NewTabLink>{" "}
+                - This week was another short break from SARS-CoV-2, and we
+                turned our attention towards genomic epidemiology of a different
+                viral pathogen - Mumps virus.
+              </span>,
+              <span key={14}>
+                <NewTabLink href="https://www.youtube.com/watch?v=lFJ_2G4u8w4">
+                  Aspen Tutorial!
+                </NewTabLink>{" "}
+                - This tutorial introduces self-serve tree building in Aspen,
+                and discusses what the different tree types are useful for.
+              </span>,
+            ]}
+          />
+        </div>
+      </>
+    );
+  };
+
+  const renderQuestions = () => {
+    return (
+      <>
+        <H3>Questions?</H3>
+        <P>
+          <NewTabLink href={ROUTES.FAQ}>Check out our FAQ</NewTabLink> for
+          general questions about Aspen, privacy, and more.
+        </P>
+      </>
+    );
+  };
+
+  return (
+    <>
+      <Head>
+        <title>Aspen | Resources</title>
+      </Head>
+      <NarrowContainer>
+        {renderIntro()}
+        {renderGenomicEpiSeries()}
+        {renderQuestions()}
+        <div style={{ height: "50px" }} />
```

Comment:
ok, actually, also what is this div about?

Reply:
this div is also in the faq, privacy, and terms pages, so i kept it to be consistent with those other pages

Reply:
It seems like this is probably for spacing. We should update all the pages to use a styled component that has a margin-bottom instead of putting in the weird div with inline styling.